### ISSUE TITLE
Input method's candidate box is not following the cursor

### ISSUE THREAD

**Issue by Bryan2333** [ONLYOFFICE/appimage-desktopeditors#59]
I'm using OnlyOffice appimage 8.1.1. 

The problem I meet is that the input methods'candidate box is not following the cursor in OnlyOffice. I think the bug https://github.com/ONLYOFFICE/DesktopEditors/issues/1102 is happening again.

Here is my system information:

Operating System: Arch Linux 
KDE Plasma Version: 6.2.1
KDE Frameworks Version: 6.7.0
Qt Version: 6.8.0
Kernel Version: 6.10.10-zen1-1-zen-cjktty (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 5800H with Radeon Graphics
Memory: 13.5 GiB of RAM
Graphics Processor: AMD Radeon Graphics

Here is the screen record: 

https://github.com/user-attachments/assets/97b84ba6-0323-43f0-97b9-fa7e66956a0b

**Comment by Bryan2333** [comment]
Yes, the issue still exists in 9.0.0. Here is the screen record below.

https://github.com/user-attachments/assets/a2fd4091-1e26-405e-8f7f-dee52c99a587

**Comment by Rita-Bubnova** [comment]
@LuckySpecialist, take a look, please.

**Comment by Bryan2333** [comment]
@LuckySpecialist 

Thanks for you reply.

No, this issue is not only relevant for appimage, debian package as well as others have the same problem.

The resolution of my laptop's builtin screen is 2880x1800 with a scaling factor of 200%. The resolution of my external screen is 2560x1440 with a scaling factor of 125%. Both screens have this problem too.

The keyboard layout in my system is English (US). The input method I'm using is Fcitx5.
